modules/qualitycontrol/ReceiveVideoController.js

```diff
@@ -185,6 +220,18 @@ export default class ReceiveVideoController {
         return this._lastN;
     }
 
+    /**
+     * Sets the assumed bandwidth bps the local participant should receive from remote participants.
+     *
+     * @param {number|undefined} assumedBandwidthBps - the new value.
+     * @returns {void}
+     */
+    setAssumedBandwidthBps(assumedBandwidthBps) {
+        if (this._receiverVideoConstraints.updateAssumedBandwidthBps(assumedBandwidthBps)) {
+            this._rtc.setReceiverVideoConstraints(this._receiverVideoConstraints.constraints);
```

Comment:
Note that the different fields in the ReceiverConstraints message can be updated independently. So you could send a message with only `assumedBandwidthBps` set instead of sending the full constraints. I don't understand the flow enough to see if it makes sense though

Reply:
AFAICT this follows the same pattern as for lastN for example.